core/src/sigverify.rs

```diff
@@ -128,7 +128,7 @@ pub fn get_packet_offsets(packet: &Packet, current_offset: u32) -> (u32, u32, u3
 
     let sig_start = current_offset as usize + sig_size;
     let msg_start = current_offset as usize + msg_start_offset;
-    let pubkey_start = msg_start + pubkey_size;
+    let pubkey_start = msg_start + 1 + pubkey_size;
```

Comment:
what's this magic #?

Reply:
`size_of(Message::num_signatures)` . I'll add a constant.

Reply:
done